packages/components/src/components/contract-card/contract-card.scss

```
@@ -228,6 +228,12 @@
     }
     &__symbol {
         margin-left: 0.4rem;
+        &--ups-downs {
+            width: 7rem;
+            @include mobile {
+                width: initial;
```

```suggestion
                width: unset;
```
@kate-deriv do you intend to unset it?

---

Width is non-inherited property and for non-inherited properties, `unset` will work the same as the `initial` value. 
+ they have similar browser support (initial 96.94% and and 96.48% for unset), so there is no difference between them in this case.